integration_tests/expr_05.py

```diff
@@ -28,5 +27,11 @@ def main0():
     b = 32771
     assert test_mod(a, b) == 2128
 
+    assert 10 | 4 == 14
+    assert -105346 | -32771 == -32769
+    assert 10 & 4 == 0
+    assert -105346 & -32771 == -105348
+    assert 10 ^ 4 == 14
+    assert -105346 ^ -32771 == 72579
```

Comment:
Why are we not testing on the cases where operands are non-constant variables? For example,

```
a = 10
b = 4
assert a | b == 14
a = -105346
b = -32771
assert a & b == -32769
```

Reply:
I will address this comment soon.  Thanks.

Reply:
So far these guys work:
```python
def main0():
  a: i32 = 10
  b: i32 = 4
  assert a & b == 0
  assert a | b == 14
  assert a ^ b == 14

main0()
```
Now we just have to deal with negative, non-constant variables, and other annotation types like `i64`, `i8`, etc.